manager/src/main/java/com/redhat/service/smartevents/manager/models/Bridge.java

```diff
@@ -1,13 +1,9 @@
 package com.redhat.service.smartevents.manager.models;
 
+import java.time.ZonedDateTime;
 import java.util.Objects;
 
-import javax.persistence.Column;
-import javax.persistence.Entity;
-import javax.persistence.NamedQueries;
-import javax.persistence.NamedQuery;
-import javax.persistence.Table;
-import javax.persistence.UniqueConstraint;
+import javax.persistence.*;
```

Comment:
We usually avoid wildcard import

Reply:
ok.. will replace it with complete package name.

Reply:
done